src/custom_requests.mli

```diff
@@ -0,0 +1,22 @@
+open Import
```

Comment:
You can use shorter names at no loss to readability:

* `Custom_request` for the module
* `Custom_request.t` for the type
* `send` instead of `send_request`.